README.md

```diff
@@ -1,51 +1,37 @@
-# Bayesian Optimization for a cuboid muscle model
+# Bayesian Optimization
 
-## Dependendcies
-OpenDiHu: Version aadd55a4 (https://github.com/opendihu/opendihu/tree/aadd55a47fde8031cc4629ba138e949d54c26661)
+## Description
+Bayesian Optimization is a method to find maxima of blackbox functions with a relatively low number of function evaluations. This repository has started as a Bachelor Thesis (https://github.com/opendihu/optimization/tree/Bachelor-thesis). It contains files for Bayesian Optimization that can be used by itself or be connected to OpenDiHu to use a simulation as a function evaluation. 
```

Comment:
in the top description you must mention opendihu. Move the bachelor thesis reference to the very end please.